collector/udp_queues_linux.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2015 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !noudp_queues
+
+package collector
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/procfs"
+)
+
+type (
+	udpQueuesCollector struct {
+		fs     procfs.FS
+		desc   *prometheus.Desc
+		logger log.Logger
+	}
+)
+
+func init() {
+	registerCollector("udp_queues", defaultDisabled, NewUDPqueuesCollector)
```

Comment:
Is there a reason to disable this by default?

Reply:
No, not really. Reduce the noise maybe. And it is also a more or less special need.

Reply:
I think this might be generally useful.

Reply:
done.